streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -505,12 +611,22 @@ public void process(final ProcessorSupplier<? super K, ? super V> processorSuppl
 
         Objects.requireNonNull(processorSupplier, "ProcessSupplier cant' be null");
         final String name = builder.newProcessorName(PROCESSOR_NAME);
+        process(processorSupplier, Named.as(name), stateStoreNames);
+    }
+
+    @Override
+    public void process(final ProcessorSupplier<? super K, ? super V> processorSupplier,
+                 final Named processorName,
```

Comment:
nit: `processorName` seems to be an inconsistent parameter name compare to other methods adding a `Named` parameter.